swagger-config.yaml

```diff
@@ -289,7 +289,7 @@ components:
     field:
       name: field
       in: query
-      description: Search by particular field
+      description: 'Search by particular field. If this parameter is not specified, default behavior will return only complaints with narratives.'
```

Comment:
are the quotes needed?

Reply:
They are not necessary for this description, no. I have made an update that removes unnecessary apostrophes from the Swagger docs. The only descriptions where I left them in are the ones where it is required to perform escape character operations